pom.xml

```diff
@@ -163,5 +162,11 @@
       <version>${system-lambda.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
This wasn't necessary in my draft - can you explain why is that new compile dependency?

Reply:
This is a maven plugin for IntelliJ IDEA that provides Annotations support, I'll remove it if it's not needed